src/config.h

```diff
@@ -157,3 +157,33 @@
 #ifndef MG_ENABLE_PROFILE
 #define MG_ENABLE_PROFILE 0
 #endif
+
+#ifndef MG_ENABLE_DRIVER_INIT
+#define MG_ENABLE_DRIVER_INIT 1
+#endif
+
+#ifndef MG_TCPIP_IP
+#define MG_TCPIP_IP 0
```

Comment:
I think we should replace this with `MG_TCPIP_IP MG_U32(0, 0, 0, 0)`. If the user wants a static IP, this way we give them a hint on how to define it